cmd/analyzer/analyzer.go

```diff
@@ -318,6 +320,24 @@ func NewService(cfg *config.AnalysisConfig) (*Service, error) {
 			return evmtokenbalances.NewMain(common.RuntimeSapphire, runtimeMetadata, sourceClient, dbClient, logger)
 		})
 	}
+	if cfg.Analyzers.EmeraldContractCode != nil {
```

Comment:
Random thought:

Does it ever make sense to not enable the "ContractCode" analyzer for EVM-like runtimes? Could probably always configure the analyzer if `cfg.Analyzers.Emerald != nil`.

Same for `EvmTokens` & `EvmTokenBalances`? Could simplify the configurations a bit this way.

Reply:
I've thought about it before too, yes. Why I haven't gone along with it so far:
- I find it handy for local debugging to have fine-grained control
- If these analyzers ever need a configurable parameter, there's won't an obvious way to put them
- It would make it so that entries under the `analyzers` YAML key no longer have a 1:1 mapping to actual analyzers, which is potentially confusing

Reply:
Good points :+1:

Maybe in future the config could be rafactred a bit so that it would look like:
```yaml
analyzers:
  emerald:
     base: ...
     evm_tokens: ...
     contract_code: ...
  sapphire:
      base: ...
      ...
```

But deferentially a really low priority, probably not even worth it, as it might just cause additional work fixing all the deployments at this time. Maybe if there are ever a lot more runtimes.